load-test/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "load-test",
+  "version": "1.0.0",
+  "description": "load test the school admin application",
```

Comment:
can you make the description more generic, as it tests both

Reply:
@GuyHarwood updated